core/controllers/payload_validator.py

```diff
@@ -36,7 +36,6 @@ def validate(
         allowed_extra_args: bool,
         allow_string_to_bool_conversion: bool = False
 ) -> Tuple[Dict[str, Any], List[str]]:
-
```

Comment:
Is this an expected change in this PR i.e. why remove this line? Ditto below.

Reply:
I think it was removed during auto formatting. I can add it back if needed